qa-tests-backend/src/test/groovy/IntegrationsTest.groovy

```diff
@@ -392,6 +392,8 @@ class IntegrationsTest extends BaseSpecification {
     @Unroll
     @Tag("BAT")
     @Tag("Notifiers")
+    // connection refused to "http://localhost:8080" on P/Z
```

Comment:
remove these comments, since these are not relevant to tests executed via upstream